### PR TITLE
Old docs to be deleted

### PR DESCRIPTION
**Pull request by michael-w-williams**
- Link to issue this resolves

Removes old out of date docs and feature docs.

- What I did

Deleted the files from my branch.

- How I did it

From command line.

- How to verify it

ls

- One line description for the changelog

Old docs to be deleted. Docs no longer relevant or outdated.

- One moving picture involving robots (not mandatory but encouraged)

Robot Smash
Bernard smashes old files.

### CODE REVIEW

File: docs/operating/logs/README.md [hunk 1]
```diff
@@ -1,40 +0,0 @@
...
-below:
-
-```sh
-$ docker run -d -p 9000:9000 --name minio -e "MINIO_ACCESS_KEY=admin" -e "MINIO_SECRET_KEY=password" minio/minio server /data
-$ docker run --privileged --link minio -e "FN_LOGSTORE_URL=s3://admin:password@minio:9000/us-east-1/fnlogs" fnproject/fnserver:latest
-```
-
-you may include any other necessary args for fnproject, this example only
-illustrates running a minio server and the required args for a functions
-server to use it.
```
Comment (rdallman) [inline docs/operating/logs/README.md]: this file should probably exist as an interim solution for OSS users. long term it's going away, but the alternative doesn't exist yet...

File: docs/definitions.md [hunk 2]
```diff
@@ -1,293 +0,0 @@
...
-            "created_at": "2017-06-02T15:31:30.597+03:00",
-            "started_at": "2017-06-02T15:31:30.597+03:00",
-            "app_name": "newapp",
-            "path": "/envsync"
-        }
-    ]
-}
-```
-
-### Pagination
```
Comment (rdallman) [inline docs/definitions.md]: just double checking we defined 'how pagination works' in the new docs somewhere? it's slightly non-standard (it's not an actual 'page number' and some users may need details to make sense of it). the rest of this file seems covered elsewhere

**Comment by michael-w-williams** [comment]
Thanks Reed. I'm on it. I'll post a comment with links once I have the files or parts of the files moved.

**Comment by michael-w-williams** [comment]
File and content have been added to the new repo:

* Pagination: <https://github.com/fnproject/docs/blob/master/fn/develop/pagination.md>
* Log Stores: <https://github.com/fnproject/docs/blob/master/fn/operate/log-stores.md>

**Pull request merged**